.circleci/config.yml

```diff
@@ -32,7 +32,7 @@ commands:
     steps:
       - run:
           name: Install git
```

Comment:
Should rename this step to Install git and openssh? Or even Install dependencies maybe

Reply:
Completely 👍